locales/en.json

```diff
@@ -141,6 +141,18 @@
       "addButton": "Add payment method",
       "addDescription": "You can add your payment methods (e.g. credit cards, bank accounts) so that you don't have to re-add them from scratch for every transaction"
     },
+    "ConfirmPayment": {
+      "header":"Confirm transaction",
+      "cancel":"CANCEL",
+      "askConfirm":"Want to proceed with the transaction?",
```

Comment:
(Do you) want to...

Reply:
perhaps just "Proceed with the transaction?" would be ok, anyway we'll ask @gvarisco to review the English copy before releasing :D